deployment/build_and_release/modules/release/main.tf

```diff
@@ -733,4 +733,286 @@ resource "google_cloudbuild_trigger" "build_recovery" {
+  name = "boot-build-${var.env}"
+  location = "global"
+
+  github {
+    owner = "transparency-dev"
+    name  = "armored-witness-boot"
+
+    push {
+      branch = var.cloudbuild_trigger_branch != "" ? var.cloudbuild_trigger_branch : null
+      tag = var.cloudbuild_trigger_tag != "" ? var.cloudbuild_trigger_tag : null
+    }
+  }
+
+  build {
+    # If the trigger is not based on `tag`, create a fake one.
+    #
+    # Unfortunately, GCB has no concept of dynamically creating substitutions or
+    # passing ENV vars between steps, so the best we can do is to create a file
+    # containing our tag in the shared workspace which other steps can inspect.
+    step {
+      name = "bash"
+      script = (
+        var.cloudbuild_trigger_tag != "" ?
+        "$TAG_NAME > /workspace/git_tag && cat /workspace/git_tag" :
+        "date +'0.0.%s-incompatible' > /workspace/git_tag && cat /workspace/git_tag"
+      )
+    }
+    ### Build the bootloader binary and upload it to GCS.
+    # Use the dockerfile to build an image containing the bootloader artifact.
+    step {
+      name = "gcr.io/cloud-builders/docker"
+      entrypoint = "bash"
+      args = [
+        "-c",
+        <<-EOT
+        docker build \
+          --build-arg=TAMAGO_VERSION=${var.tamago_version} \
+          --build-arg=GIT_SEMVER_TAG=$(cat /workspace/fake_tag) \
+          --build-arg=LOG_ORIGIN=${var.origin_prefix}/${var.log_shard} \
+          --build-arg=LOG_PUBLIC_KEY=${var.log_public_key} \
+          --build-arg=OS_PUBLIC_KEY1=${var.os_public_key1} \
+          --build-arg=OS_PUBLIC_KEY2=${var.os_public_key1} \
+          --build-arg=BEE=${var.bee} \
+          --build-arg=CONSOLE=${var.console} \
+          -t builder-image \
+          .
+        EOT
+      ]
+    }
+    # Prepare a container with a copy of the artifacts.
+    step {
+      name = "gcr.io/cloud-builders/docker"
+      args = [
+        "create",
+        "--name",
+        "builder_scratch",
+        "builder-image",
+      ]
+    }
+    # Copy the artifacts from the container to the Cloud Build VM.
+    step {
+      name = "gcr.io/cloud-builders/docker"
+      args = [
+       "cp",
+       "builder_scratch:/build",
+       "output",
+      ]
+    }
+    # List the artifacts.
+    step {
+      name = "bash"
+      script = "ls output"
+    }
+    # Copy the artifacts from the Cloud Build VM to GCS.
+    step {
+      name = "gcr.io/cloud-builders/gcloud"
+      entrypoint = "bash"
+      args = [
+        "-c",
+        <<-EOT
+        gcloud storage cp \
+          output/armored-witness-boot.imx \
+          gs://${var.firmware_bucket_prefix}-${var.log_shard}/$(sha256sum output/armored-witness-boot.imx | cut -f1 -d" ")
+        EOT
+      ]
+    }
+    # HAB: Create SRK table & hash
+    # TODO(al): we should probably store the generated SRK/hash in a GCS bucket
+    # and then compare each time to ensure that nothing bad has happened with
+    # our PKI.
```

Comment:
We can remove this comment now (the "Assert SRK hash value" step essentially does this)